.graphqlrc.yml

```diff
@@ -0,0 +1,13 @@
+projects:
+  # 'app-core':
```

Comment:
still need these commented out things?

Reply:
Yeah, just shows people how to quickly swap to local dev. I couldn't figure out how to do it with a command